### PR TITLE
Added coverage, cov thresholds

### PR DESCRIPTION
**Pull request by spiray**
I added coverage and coverage thresholds for the api directory. The tests will now fail if the coverage goes below what it is set at now.

### CODE REVIEW

**Comment by ewenjo** [comment]
I feel like this change can be counter productive and cause unwanted problems. If someone ever creates a new API route without tests, the total coverage will most likely go bellow the threshold and fail the tests. 

It's already close to the threshold since we haven't created tests for edge cases and the admin route.

What do you think? @spiray

**Comment by spiray** [comment]
Should we keep them at 70 so we know we are mostly covered but is a safe distance from new features causing it to fail?

**Comment by ewenjo** [comment]
That would be better than what it currently is, but still, I feel like we should wait with this until we have finished writing tests for the admin route and the helper functions.

What I'm saying is, I don't want new contributors to worry about code coverage and testing when they're contributing.

**Comment by spiray** [comment]
On the other hand that is what it is for, We don't want untested endpoints coming in... For now the coverage is only measuring the api directory.

**Comment by ewenjo** [comment]
Okay, I get that. So, I think we should fix most/all of the uncovered lines before merging this and also make sure that all new endpoints in the future are tested/covered before they're merged.

**Comment by spiray** [comment]
@ewenjo See updates.
image

File: src/api/admin/admin.test.js [hunk 1]
```diff
@@ -10,3 +10,11 @@ describe('GET /admin', () => {
       .expect(404, done);
   });
 });
+
+describe('GET /admin/seed', () => {
```
Comment (ewenjo) [inline src/api/admin/admin.test.js]: Should be `POST /admin/seed`

File: package.json [hunk 2]
```diff
@@ -1,60 +1,72 @@
...
+    "main": "src/index.js",
+    "jest": {
+        "testEnvironment": "node",
+        "preset": "jest-puppeteer",
+        "collectCoverageFrom": [
+            "!**node_modules**",
+            "src/api/**/*.js"
+        ],
+        "coverageThreshold": {
+            "global": {
```
Comment (ewenjo) [inline package.json]: Thresholds can be higher than 70, as long as we make sure that future contributions are tested/covered.

**Comment by tejas-hosamani** [comment]
@spiray This is a minor thing but I think the tab-space is 4. This is updating the entire file/s. Example, package.json

Also, I just learned about Code coverage because of this PR - it blew my mind!! This stuff is amazing. I am totally geeking about it.

File: package.json (hunk 2 above)
Comment (spiray) [reply package.json]: Updated

**Review by ewenjo — APPROVED** [review]

**Review by tejas-hosamani — APPROVED** [review]
